storage/google/cloud/storage/client.py

```diff
@@ -561,6 +562,66 @@ def list_buckets(
             extra_params=extra_params,
         )
 
+    def create_hmac_key(self, service_account_email):
```

Comment:
@tseaver could you make project_id an optional parameter to all hmac methods? By default it will use credentials project_id.

Reply:
Done: 58fed57